src/components/LoanCards/BorrowerInfo.vue

```diff
@@ -17,6 +17,7 @@
 				v-kv-track-event="['Lending', 'click-Read more', 'Read more', id, 'true']">
 				Read more
 			</router-link>
+			<div>{{ loanLength }} months</div>
```

Comment:
The comp in Vue-22 shows some bolded copy before the loan term.

Reply:
👍